saleor/tests/fixtures.py

```diff
@@ -4223,3 +4224,102 @@ def app_export_event(app_export_file):
         app=app_export_file.app,
         parameters={"message": "Example error message"},
     )
+
+
+@pytest.fixture
```

Comment:
As I explained above. No need to extract it to the global fixtures file.  (Yes I know that we did this with Vatlayer, but this is specific case which we're going to resolve in the future)

Reply:
AFAIK that almost 50 test cases are using this fixture. How should I refactor that? Move that fixture into each module conftests?

Reply:
It should be used with sendgird's tests, so it should be placed there in the conftest